### PR TITLE
OD-158 [Feature] Added chat icon to the Directory and Featured list layouts

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-158

## Description
 Added chat icon to the Directory and Featured list layouts

## Screenshots/screencasts
https://share.getcloudapp.com/7KuoBrzk

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

## Notes
To work correctly needs this [PR](https://github.com/Fliplet/fliplet-theme-default/pull/168) as well

### CODE REVIEW

File: js/default-configs/layouts-config.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ window.flListLayoutConfig = {
   'small-h-card': {
     'filtersEnabled': false,
     'filterFields': [],
-    'chatEnabled': true,
+    'chatEnabled': false,
```
Comment (tonytlwu) [inline js/default-configs/layouts-config.js:224]: Why is this `false`?

Comment (YaroslavOvdii) [reply js/default-configs/layouts-config.js:224]: @tonytlwu I thought that we want to add this option as we did with sorting, allowing users to active it by themself.
Should we enable the chat option from the start?

Comment (tonytlwu) [reply js/default-configs/layouts-config.js:224]: I see. Then yes, `false` is fine 🆗

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: LFD Directory and Featured List layouts
Verify: Detail views are correctly rendered and functioning as expected

Squash and merge: Use PR title

**Pull request merged**